src/Elrond/Entry.h

```diff
@@ -14,7 +14,7 @@ namespace TW::Elrond {
 /// Note: do not put the implementation here (no matter how simple), to avoid having coin-specific includes in this file
 class Entry final: public CoinEntry {
 public:
-     bool validateAddress(TWCoinType coin, const std::string& address, TW::byte p2pkh, TW::byte p2sh, const char* hrp) const;
+    bool validateAddress(TWCoinType coin, const std::string& address, const PrefixVariant& addressPrefix) const;
```

Comment:
```suggestion
     bool validateAddress(TWCoinType coin, const std::string& address, const PrefixVariant& addressPrefix) const;
```

Reply:
There are a few other places, maybe we can enable auto format now

Reply:
Auto-format after files are already pushed doesn't work in my IDE, but I fixed the above place^